docs/terraform-provider-documentation

```diff
@@ -0,0 +1,3 @@
+# Terraform provider documentation
```

Comment:
If you want to display the page in docs.airbyte.io you must add the documentation in sidebar.js inside docusarus page.

Reply:
I think the documentation can be placed inside manage-airbyte folder